app/models/pipeline_run_stage.rb

```diff
@@ -123,26 +127,21 @@ def due_for_aegea_check?
   end
 
   def update_job_status
-    if !id || !started? || succeeded? || failed?
-      # This is called only from PipelineRun.update_status and only when the guard above is false.
+    if !id || !started?
       Airbrake.notify("Invalid precondition for PipelineRunStage.update_job_status #{id} #{job_id} #{job_status}.")
       return
     end
-    if checked?
-      # This can happen due to overly-eager frequent calls from pipeline_monitor through PipelineRun.update_job_status.
-      Rails.logger.info "Job #{id} #{job_id} checked and waiting to load results."
+    if failed?
+      terminate_job
       return
     end
-    if output_ready?
-      self.job_status = STATUS_CHECKED
-      # save before enqueue, to prevent minor race
-      save
-      Resque.enqueue(LoadResultForRunStage, id)
```

Comment:
Where did this move to?

Reply:
to monitor_results in the pipeline_run model